<?xml version="1.0"?>
<launch>
  <param name="robot_ip" value="10.224.20.198" />
  <param name="publish_command" value="true" />

  <!-- Run control node-->
  <node name="franka_control" type="franka_control" pkg="pick_and_place_libfranka" output="screen" />


</launch>